### PR TITLE
Added :param to the _get_exc_class_and_code method docstring

### LINKED ISSUE #3252: Should _get_exc_class_and_code method have param entry for the docstring?
Currently the `_get_exc_class_and_code` method does not have `:param:` entry for `exc_class_or_code`.

If so, I can add `:param:` entry for methods which does not contain `:param:` entry.

https://github.com/pallets/flask/blob/d4b688bd035b9704b3168b28fed39c8fcfe3b997/src/flask/app.py#L1334-L1346

### PR DESCRIPTION
**Pull request by aligoren**
Describe what this patch does to fix the issue.

Added :param to the _get_exc_class_and_code method docstring

Link to any relevant issues or pull requests.

This fixes #3252

### CODE REVIEW

File: src/flask/app.py [hunk 1]
```diff
@@ -1332,7 +1332,10 @@ def decorator(f):
 
     @staticmethod
     def _get_exc_class_and_code(exc_class_or_code):
-        """Ensure that we register only exceptions as handler keys"""
+        """Ensure that we register only exceptions as handler keys
+        
+        :param exc_class_or_code: the class for the or exception code as integer
```
Comment (jab) [inline src/flask/app.py]: Typo?

Comment (aligoren) [reply src/flask/app.py]: Okay. I changed. Sorry for the mistake.

**Comment by jab** [comment]
CI failed but looks like for unrelated reasons. Inclined to merge this.

File: src/flask/app.py [hunk 2]
```diff
@@ -1332,7 +1332,11 @@ def decorator(f):
 
     @staticmethod
     def _get_exc_class_and_code(exc_class_or_code):
-        """Ensure that we register only exceptions as handler keys"""
+        """Ensure that we register only exceptions as handler keys
+
+        :param exc_class_or_code: the class for the exception, or
```
Comment (davidism) [inline src/flask/app.py]: "Any exception class, or an HTTP status code as an integer.

Comment (davidism) [reply src/flask/app.py]: Wrap at 72 characters with one indent level (I know the existing docs aren't consistent about this).

**Pull request merged**